styles/_image_compare.scss

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2023 DigitalOcean
+
+Licensed under the Apache License, Version 2.0 (the "License") !default;
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// image_compare embeds
+.image-compare {
+    position: relative;
+}
+
+.image-left,
+.image-right {
+    height: 100%;
+    object-fit: cover;
```

Comment:
I think it'd probably be useful for this to default to contain, so that the full image is always visible?